openstack_controller/assets/configuration/spec.yaml

```diff
@@ -181,21 +243,95 @@ files:
             example: false
             display_default: false
             type: boolean
-        - template: instances/default
-          overrides:
-            service.hidden: true
-            min_collection_interval.hidden: true
-            empty_default_hostname.hidden: true
-        - template: instances/http
-          overrides:
-            tls_use_host_header.hidden: true
-            auth_type.hidden: true
-            use_legacy_auth_encoding.hidden: true
-            auth_token.hidden: true
-            aws_region.hidden: true
-            aws_host.hidden: true
-            aws_service.hidden: true
-            extra_headers.hidden: true
+        - name: endpoint_interface
+          description: |
+           Indicates the visibility scope of the component's endpoints to use. The possible values are:
+             public: This endpoint is accessible from outside the cloud and typically denotes a public-facing URL or 
+                 IP address.
+             internal: This endpoint is typically used for communication between services within the cloud 
+                 infrastructure itself.
+                 It may be an internal IP or hostname that's not accessible from outside the cloud.  
+             admin: This endpoint is specifically for administrative tasks and may have extended privileges or 
+                 functionality compared to the public and internal endpoints.
+                 It's meant for cloud operators or administrators.
+          value:
+            example: "public"
```

Comment:
Question: What is improved with this suggestion?

Reply:
It demonstrates that "public" is the default value, but make "internal" the example, since users who enable this option are most likely to change it to "internal," not "public"